### PR TITLE
1.14 patches

### PR DESCRIPTION
**Pull request by misode**
Fixes most of the 1.14 issues:

- standard crafing
   - rename rose_red to red_dye
   - new stair type recipes need to be added
   - rename stone_stab to smooth_stone_slab for crafting smooth_stone
- master crafting
   - escape single quote in "Mastercraftsman\'s Bench" (throws error)
- desire lines
   - working, but I noticed creative players don't replace grass. is this intended?
- disassembers
   - armor stand doesn't have the gm4_no_edit tag
- ender hoppers
   - the item detection region seems to be shifted +0.5 blocks in the +X and +Z direction
- lightning rods
   - unbalanced quotes in advancements
- record crafting
   - use new dye item id's
- mob conversions
   - limited to only one item each cycle (see above)
- scuba gear
   -  add new sign types
- shroomites
   -  add new sign types
- sunken treasure
   - fix broken json in lore in messages in a bottle

### CODE REVIEW

**Comment by Bluefire-610** [comment]
Was this tested in game?

**Comment by AndanteDevs** [comment]
I tested the changes in game, and didn't experience any issues

File: gm4_trapped_signs/pack.mcmeta [hunk 1]
```diff
@@ -4,7 +4,6 @@
     "description":"A Gamemode 4 Module"
   },
   "module_name":"Trapped Signs",
-  "module_id":"trapped_signs",
```
Comment (Bluefire-610) [inline gm4_trapped_signs/pack.mcmeta]: Why were the "module_id": fields removed? Does the website not read those anymore?

Comment (AndanteDevs) [reply gm4_trapped_signs/pack.mcmeta]: I believe that's because, at least for orbis modules, they haven't been updated yet.

**Comment by Bluefire-610** [comment]
@sparks0 Do we want the diamond recipe removal in here? On Toffee's PR #179 you said you wanted to leave them in but comment them out.

**Comment by Kroppeb** [comment]
If you want to readd them you could just revert the commit that removed those

File: gm4_trapped_signs/pack.mcmeta (hunk 1 above)
Comment (misode) [reply gm4_trapped_signs/pack.mcmeta]: Removing that field hides the module on the site.

**Comment by misode** [comment]
> Was this tested in game?

Yes, all modules have been tested using Andante's Lore fix PR. All modules enabled in this branch are working in 1.14

**Comment by misode** [comment]
> Do we want the diamond recipe removal in here? On Toffee's PR #179 you said you wanted to leave them in but comment them out.

#179 is targeting 1.13

**Pull request merged**